plugins/org.jkiss.dbeaver.tasks.ui.view/src/org/jkiss/dbeaver/tasks/ui/view/TaskHandlerRun.java

```diff
@@ -85,4 +93,25 @@ public void updateElement(UIElement element, Map parameters) {
         }
 
     }
+
+    private static boolean promptTaskVariables(@NotNull DBTTask task) {
```

Comment:
I'd prefer to call it like `confirmTaskVariables` as we stop task execution if user press Cancel or closed the dialog.

Reply:
It's sounds a bit odd to me but okay.